src/raven.js

```diff
@@ -748,35 +748,41 @@ function send(data) {
     // Set lastEventId after we know the error should actually be sent
     lastEventId = data.event_id || (data.event_id = uuid4());
 
-    makeRequest(data);
-}
+    logDebug('debug', 'Raven about to send:', data);
 
+    if (!isSetup()) return;
 
-function makeRequest(data) {
-    var img,
-        src;
+    var url = globalServer + authQueryString;
```

Comment:
auth should def be something passed into the callback and the default callback would just be responsible for changing the querystring